### PR TITLE
Docs update for set dev batching size to 1

### PR DESCRIPTION
**Pull request by alex-tpom6oh**

### CODE REVIEW

File: pages/implementation/avo-inspector-sdk-reference.mdx [hunk 1]
```diff
@@ -43,11 +43,10 @@ Find the overview in <Link passHref href="/implementation/devs-101#avo-inspector
 
 ## <a name="batching"></a> Batching
 
-In order to ensure our SDK doesn't have a large impact on performance or battery life it supports event schemas batching. Default values are 30 schemas per batch and smaller batches are sent if more than 30 seconds pass between uploads.
+In order to ensure our SDK doesn't have a large impact on performance or battery life it supports event schemas batching. In production, default values are 30 schemas per batch and smaller batches are sent if more than 30 seconds pass between uploads.
+In development every event is sent as soon as it's provided to the Inspector by default.
```
Comment (alex-tpom6oh) [inline pages/implementation/avo-inspector-sdk-reference.mdx:47]:
```suggestion
In development batching is disabled by default.
```

File: pages/implementation/devs-101.mdx [hunk 2]
```diff
@@ -304,6 +304,12 @@ nested objects
 
 > Bonus: this docs you are reading <a target="_blank" rel="noopener noreferrer" href="https://github.com/avohq/docs">are open sourced</a>, if you find anything wrong or missing, please open a pull request or an issue!
 
+#### <a name="batching"></a> Batching
+
+In order to ensure our SDK doesn't have a large impact on performance or battery life it supports event schemas batching. In producition, default values are 30 schemas per batch and smaller batches are sent if more than 30 seconds pass between uploads.
+In development events are sent as soon as they are provided to the Inspector by default.
```
Comment (alex-tpom6oh) [inline pages/implementation/devs-101.mdx:310]:
```suggestion
In development batching is disabled by default.
```

File: pages/implementation/inspector/sdk/android.mdx [hunk 3]
```diff
@@ -235,7 +235,8 @@ static public void setBatchSize(int newBatchSize) // static method on AvoInspect
 fun setBatchSize(newBatchSize: Int) // static method on AvoInspector
 ```
 
-Enables manual control over events batching. Default batch size in production is 30, i.e. the library attempts to send event schemas to the server when it has 30 or more schemas.
+Enables manual control over events batching. Default batch size in production is 30, i.e. the library attempts to send event schemas to the server when it has 30 or more schemas. 
+In development the events are sent as soon as they are provided to the Inspector by default.
```
Comment (alex-tpom6oh) [inline pages/implementation/inspector/sdk/android.mdx:239]:
```suggestion
In development batching is disabled by default.
```

File: pages/implementation/inspector/sdk/android.mdx [hunk 4]
```diff
@@ -252,7 +253,6 @@ fun setBatchFlushSeconds(newBatchFlushSeconds: Int) // static method on AvoInspe
 ```
 
 Enables manual control over events batching. Default production batch flush interval is 30 seconds, i.e. the library attempts to send event schemas to the server when 30 or more seconds pass, given there are unsent schemas.
```
Comment (alex-tpom6oh) [inline pages/implementation/inspector/sdk/android.mdx:255]:
```suggestion
Enables manual control over events batching. Default production batch flush interval is 30 seconds, i.e. the library attempts to send event schemas to the server when 30 or more seconds pass, given there are unsent schemas.
In development batching is disabled by default.
```

File: pages/implementation/inspector/sdk/ios.mdx [hunk 5]
```diff
@@ -217,6 +217,7 @@ func setBatchSize(newBatchSize: Int32)
 
 This is a class method.
 Enables manual control over events batching. Default batch size in production is 30, i.e. the library attempts to send event schemas to the server when it has 30 or more schemas.
+In development the events are sent as soon as they are provided to the Inspector by default.
```
Comment (alex-tpom6oh) [inline pages/implementation/inspector/sdk/ios.mdx:220]:
```suggestion
In development batching is disabled by default.
```

File: pages/implementation/inspector/sdk/react-native.mdx [hunk 6]
```diff
@@ -162,6 +162,7 @@ setBatchSize(newBatchSize: number): void;
 ```
 
 Enables manual control over events batching. The default batch size in production is 30, i.e. the library attempts to send event schemas to the server when it has 30 or more schemas.
+In development the events are sent as soon as they are provided to the Inspector by default.
```
Comment (alex-tpom6oh) [inline pages/implementation/inspector/sdk/react-native.mdx:165]:
```suggestion
In development batching is disabled by default.
```

File: pages/implementation/inspector/sdk/web.mdx [hunk 7]
```diff
@@ -162,6 +162,7 @@ setBatchSize(newBatchSize: number): void;
 ```
 
 Enables manual control over events batching. The default batch size in production is 30, i.e. the library attempts to send event schemas to the server when it has 30 or more schemas.
+In development the events are sent as soon as they are provided to the Inspector by default.
```
Comment (alex-tpom6oh) [inline pages/implementation/inspector/sdk/web.mdx:165]:
```suggestion
In development batching is disabled by default.
```

**Comment by hugihlynsson** [comment]
#### 📝Changed routes:


| [**…nspector-sdk-reference**](https://docs-93aqvlkrw.avo-dev.app/implementation/avo-inspector-sdk-reference) | [**/implementation/devs-101**](https://docs-93aqvlkrw.avo-dev.app/implementation/devs-101) |
|:-:|:-:|
|<a href="https://docs-93aqvlkrw.avo-dev.app/implementation/avo-inspector-sdk-reference">Screenshot of /implementation/avo-inspector-sdk-reference</a><br /><sup><a href="https://images.weserv.nl?url=https://iad.microlink.io/pfW9RGhNrxSpyWx83zB6fh2UeYgIkmuT4I7uijqKw3lkP_D_NLolpQCYtiu2S1H1XMRYvkPsqKhOQ8Nfevrvjw.png">(view full size)</a> |<a href="https://docs-93aqvlkrw.avo-dev.app/implementation/devs-101">Screenshot of /implementation/devs-101</a><br /><sup><a href="https://images.weserv.nl?url=https://iad.microlink.io/nHQvfk8LmnfTU7XYKh1WAdtSwXFs-RbQ4bE9l37e8m2J3JP9p8LAHjFmzvd9cSJWVVNW-r3U_8pJwK-pPgHPBA.png">(view full size)</a> |



| [**…/inspector/sdk/android**](https://docs-93aqvlkrw.avo-dev.app/implementation/inspector/sdk/android) | [**…tion/inspector/sdk/ios**](https://docs-93aqvlkrw.avo-dev.app/implementation/inspector/sdk/ios) |
|:-:|:-:|
|<a href="https://docs-93aqvlkrw.avo-dev.app/implementation/inspector/sdk/android">Screenshot of /implementation/inspector/sdk/android</a><br /><sup><a href="https://images.weserv.nl?url=https://iad.microlink.io/HYtvLEPnP8HmjWQqcdjRKetS5Bv2aB_oguvuD4Qjy1bQRqpz5zeMAR4-6zy7Kkdt2s7cvaoyZV3tDf_bJ3Yt1g.png">(view full size)</a> |<a href="https://docs-93aqvlkrw.avo-dev.app/implementation/inspector/sdk/ios">Screenshot of /implementation/inspector/sdk/ios</a><br /><sup><a href="https://images.weserv.nl?url=https://iad.microlink.io/2VKrFnIeBuiKzHNG3wzKhDCouOAbHNv4LQ57GQDciEBn37FR1SYZsmW0BRbQvnq6Y8Rwnm5PaAGBCJuPvyPN1w.png">(view full size)</a> |



| [**…ector/sdk/react-native**](https://docs-93aqvlkrw.avo-dev.app/implementation/inspector/sdk/react-native) | [**…tion/inspector/sdk/web**](https://docs-93aqvlkrw.avo-dev.app/implementation/inspector/sdk/web) |
|:-:|:-:|
|<a href="https://docs-93aqvlkrw.avo-dev.app/implementation/inspector/sdk/react-native">Screenshot of /implementation/inspector/sdk/react-native</a><br /><sup><a href="https://images.weserv.nl?url=https://iad.microlink.io/PbEQMwMy728MuYE8hnmIouyhdwZVLQqKzKXvceXQt0ZsAeCOGY3jGRUO3HWip5dv-mJ-Zy6NLUOIjBcr438iZw.png">(view full size)</a> |<a href="https://docs-93aqvlkrw.avo-dev.app/implementation/inspector/sdk/web">Screenshot of /implementation/inspector/sdk/web</a><br /><sup><a href="https://images.weserv.nl?url=https://iad.microlink.io/ZxLC7gMIkmNP2lepislA2gZnOO1320Fk_Gar0l3HLvca1BchLrBVc3WKw2TWy9oHNuN99nve0hsz_iAgiyb6mQ.png">(view full size)</a> |



And 1 other route:
- [**/implementation/setup-inspector-sdk**](https://docs-93aqvlkrw.avo-dev.app/implementation/setup-inspector-sdk)



Commit 36d2e3c0acb467a66b4beb4a8d6a560736994a47 (https://docs-93aqvlkrw.avo-dev.app).

**Review by logason — APPROVED** [review]

File: pages/implementation/setup-inspector-sdk.mdx [hunk 8]
```diff
@@ -7,15 +7,6 @@ import Link from '../../components/Link';
...
-And the concept batching used in the SDKs.
-
-#### <a name="batching"></a> Batching
-
-In order to ensure our SDK doesn't have a large impact on performance or battery life it supports event schemas batching. Default values are 30 schemas per batch and smaller batches are sent if more than 30 seconds pass between uploads.
-You can change the values using `AvoInspector.setBatchSize(newBatchSize: Int)` and `AvoInspector.setBatchFlustSeconds(newBatchFlushSeconds: Int)` class methods to fine tune the library performance.
-
-> When integrating the Avo Inspector for the first time and sending the first event it might not appear right away because of batching. Be sure to trigger a few events within a few seconds to make the batch go out to Avo servers. It will be reflected in the logs.
-
 ### Impelmentation
```
Comment (logason) [inline pages/implementation/setup-inspector-sdk.mdx:10]:
```suggestion
### Implementation
```

**Pull request merged**